frontend/www/js/omegaup/components/course/Statistics.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="container-fluid">
+    <h2 class="text-center">
+      <a v-bind:href="courseUrl">{{ course.name }}</a>
```

Comment:
nit:

```suggestion
      <a v-bind:href="`/course/${course.alias}/`">{{ course.name }}</a>
```

it might be better to inline this, since it's only used once.